ovn-tester/ovn_tester.py

```diff
@@ -129,6 +132,12 @@ def read_config(configuration):
             n_pods_per_node=bringup_args.get('n_pods_per_node', 10)
         )
 
+        light_density_args = config.get('light_density', dict())
```

Comment:
Nit: I would call this `density_light_*` everywhere as we'll also have `density_heavy_*` in the future.